test/cpp/test_aten_xla_tensor_1.cpp

```diff
@@ -467,6 +467,18 @@ TEST_F(AtenXlaTensorTest, TestDot) {
   });
 }
 
+TEST_F(AtenXlaTensorTest, TestDotInt64) {
+  torch::Tensor a = torch::randint(0, 100, {10});
+  torch::Tensor b = torch::randint(0, 100, {10});
+  torch::Tensor c = torch::dot(a, b);
+  ForEachDevice([&](const torch::Device& device) {
+    torch::Tensor xla_a = CopyToDevice(a, device);
+    torch::Tensor xla_b = CopyToDevice(b, device);
+    torch::Tensor xla_c = torch::dot(xla_a, xla_b);
+    AllClose(c, xla_c);
+  });
+}
```

Comment:
can we add counter checks similar to 
```
  ExpectCounterNotChanged("aten::.*", cpp_test::GetIgnoredCounters());
  ExpectCounterChanged("xla::mm", cpp_test::GetIgnoredCounters());
```
we do expect `aten` counter to change since it will fallback, but xla counter will also changed since it will first dispatched to XLA.